AUTHORS

```diff
@@ -20,6 +20,7 @@ liuhuahang <liuhuahang@zerus.co>
 David Mankin <dmankin@atlassian.com>
 Kamal Marhubi <kamal@marhubi.com>
 Carl Mastrangelo <carl-mastrangelo@users.noreply.github.com>
+Fabian Meumertzheim <fabian@meumertzhe.im>
```

Comment:
Please do not add yourself to AUTHORS.
By agreeing to the CLA you relinquish copyright claim.

Reply:
Done